syntax = "proto3";

package partner;

// Request
message GetPartnerRequest {
  string id = 1;
}

// Response
message GetPartnerResponse {
  string id = 1;
  string name = 2;
  bool active = 3;
  string tenant_id = 4;
}

// Service definition
service PartnerService {
  rpc GetPartner (GetPartnerRequest) returns (GetPartnerResponse);
}
